### PR TITLE
Dev wiz1989

### PR DESCRIPTION
**Pull request by Sphyralg**
changes include:

    added Mon Alomomola
    form change logic
    added move Heart Swap
    new move effect Heart_Swap to trigger the change
    only trigger evolution once per Alomomola battle
    added Cmd_callnative to call C functions from battle_Script_commands

### CODE REVIEW

File: graphics/pokemon/alomomola/back.png
Comment (jogaklaa) [inline graphics/pokemon/alomomola/back.png:1]: should this be the shiny sprite?

File: src/strings.c [hunk 1]
```diff
@@ -1056,6 +1056,7 @@ const u8 gText_BerryCrush2[] = _("BERRY CRUSH");
 const u8 gText_PressingSpeedRankings[] = _("Pressing-Speed Rankings");
 const u8 gText_Var1Players[] = _("{STR_VAR_1} PLAYERS");
 ALIGNED(4) const u8 gText_Ghost[] = _("GHOST");
+ALIGNED(4) const u8 gText_Luvdisc[] = _("LUVDISC");
```
Comment (Sphyralg) [inline src/strings.c]: Is this variable necessary?
Outside of a commented out line I don't see it used anywhere.

Comment (wiz1989) [reply src/strings.c]: good catch. This one is actually no longer required. Was part of an earlier implementation approach.

File: graphics/pokemon/alomomola/back.png
Comment (wiz1989) [reply graphics/pokemon/alomomola/back.png:1]: no. This is the back sprite, that is shown when you have the mon on your side of the battle.
The shininess comes only from the shiny.pal file.

**Review by wiz1989 — APPROVED** [review]
unnecessary string was deleted after the last review. Looks good to go now.

**Pull request merged**